website/docs/guides/scripts.md

```diff
@@ -185,9 +185,13 @@ chmod +x script.sc
 ```
 
 ```text
```

Comment:
````suggestion
```bash
````

Reply:
It is outpu from the previous command, so there's no need to execute the script again.

Reply:
ahhhh, ok, my bad